doc/source/conf.py

```diff
@@ -91,7 +91,8 @@
 
 redirects = {
      "user_guide/getting_started_with_dpf_server": "../getting_started/dpf_server.html",
-     "concepts/index": "../user_guide/index.html#concepts"
+     "concepts/index": "../user_guide/index.html#concepts",
+     "contributing": "source/getting_started/contributing.html"
```

Comment:
```suggestion
     "contributing": "getting_started/contributing.html"
```
Isn't that the correct new path?